app/src/main/java/nextstep/shoppingcart/component/ProductItem.kt

```diff
@@ -0,0 +1,86 @@
+package nextstep.shoppingcart.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import nextstep.shoppingcart.ui.theme.ShoppingCartTheme
+import java.text.NumberFormat
+import java.util.Locale
+
+@Composable
+fun ProductItem(
+    modifier: Modifier = Modifier,
+    name: String,
+    imageUrl: String,
+    price: Long
+) {
+    val formattedAmount = remember(price) {
+        NumberFormat.getNumberInstance(Locale.KOREA).format(price) + "원"
+    }
+    Column(
+        modifier
+            .wrapContentWidth()
+            .wrapContentHeight()
+    ) {
```

Comment:
```suggestion
    Column {
```
기본적으로 wrapContent로 동작하기에 지정하지 않으셔도 됩니다.